frontend/src/views/OrderView.vue

```diff
@@ -1,74 +1,183 @@
-      <p class="font-bold">Daten</p>
-      <FloatLabel>
-        <InputText id="constructionProject" class="w-full" />
-        <label for="constructionProject">Bauvorhaben</label>
-      </FloatLabel>
-      <FloatLabel>
-        <Dropdown v-model="status" :options="orderStatusTypes" class="w-full" id="select" />
-        <label for="select">Status</label>
-      </FloatLabel>
-      <p class="font-bold">Skonto</p>
-      <div class="grid grid-cols-3 gap-1 mt-3">
-        <FloatLabel>
-          <Dropdown
-            v-model="discount"
-            :options="discountChoice"
-            class="w-full"
-            id="selectDiscount"
-          />
-          <label for="selectDiscount">Skontoberechtig</label>
-        </FloatLabel>
-        <FloatLabel>
-          <Dropdown
-            v-model="discountPeriod"
-            :options="discountPeriodChoice"
-            class="w-full"
-            id="select"
-          />
-          <label for="select">Skontodauer</label>
-        </FloatLabel>
-        <FloatLabel>
-          <InputText id="percent" class="w-full" />
-          <label for="percent">Skonto(%)</label>
-        </FloatLabel>
-      </div>
-      <div>
-        <label for="description" class="w-full my-3">Beschreibung</label>
-        <Textarea v-model="decription" rows="5" class="w-full" id="description" />
-      </div>
-      <div class="my-1">
-        <p class="font-bold">Kunde</p>
-        <Button class="my-2" type="button" label="Kunden Zuordnen" severity="secondary"></Button>
-      </div>
+    <div class="flex flex-row justify-between mb-3">
+      <Button
+        @click="onOrdersList"
+        icon="pi pi-arrow-left"
+        size="small"
+        severity="secondary"
+        text
+        raised
+      />
+      <Button
+        @click="onSaveOrder"
+        type="button"
+        label="Auftrag Speichern"
+        :disabled="isButtonDisabled"
+      ></Button>
+      <Button v-if="isEditing" type="button" label="Löschen" severity="danger" text raised></Button>
     </div>
+    <Card>
+      <template #content>
+        <div class="flex flex-col gap-y-5">
+          <p class="font-bold">Daten</p>
+          <FloatLabel>
+            <InputText id="constructionProject" v-model="orderInfo.title" class="w-full" />
+            <label for="constructionProject">Bauvorhaben</label>
+          </FloatLabel>
+          <FloatLabel>
+            <Dropdown
+              v-model="orderInfo.status"
+              :options="orderStatusTypes"
+              class="w-full mt-3"
+              id="select"
+            />
+            <label for="select" class="mt-3">Status</label>
+          </FloatLabel>
+          <p class="font-bold">Skonto</p>
+          <div class="grid grid-cols-2 gap-1 mt-3">
+            <FloatLabel>
+              <Dropdown
+                v-model="orderInfo.can_have_cash_discount"
+                :options="discountChoice"
+                class="w-full"
+                id="selectDiscount"
```

Comment:
HTML properties should use kebap-case, no?

```suggestion
                id="select-discount"
```